xml/System/Index.xml

```diff
@@ -251,10 +272,24 @@
         <Parameter Name="length" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="length">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="length">The length of the collection that the Index will be used with. Must be a positive value.</param>
+        <summary>Calculate the offset from the start using the giving collection length.</summary>
+        <returns>The offset.</returns>
+        <remarks>
+          <format type="text/markdown">
```

Comment:
This remark was a bit difficult to understand, especially the 3rd sentence, so I left it untouched. Can I get some help to properly rewrite it?
There's another similar remark in another API in this same PR.

Reply:
Ok I gave it a try.

Reply:
I still don't think it's clear. There's nothing that ties `length` to the length of a collection. And the source ignores the length argument if the index is from the start and just returns the original index:  
```
        public int GetOffset(int length)
        {
            int offset = _value;
            if (IsFromEnd)
            {
                // offset = length - (~value)
                // offset = length + (~(~value) + 1)
                // offset = length + value + 1
 
                offset += length + 1;
            }
            return offset;
        }
```

Reply:
Help us, @tarekgh, you're our only hope.
Can you help clarify this remark?

Reply:
What I understood from Tarek's explanation is that length is only considered when retrieving the offset from the end. You are correct, the method ignores the length if it's from the start.